### ISSUE TITLE
Notify when a timeout loop is detected

### ISSUE THREAD

**Issue by aronisstav** [parapluu/Concuerror#83]
```
[...]
261: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 262: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 263: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 264: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 265: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 266: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 267: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
 268: P.1.1.1: receive timeout expired after 500 ms
    in gen_fsm.erl line 425
[...]
```

This kind of trace can be detected and a notice about the problem should be emitted.

**Comment by aronisstav** [comment]
Special case of #14

**Comment by aronisstav** [comment]
Should be fixed together with #14. Closing.
